spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe User, type: :model do
+  describe "presence" do
+    it { should validate_presence_of(:name)}
+    it { should validate_presence_of(:email)}
+    it { should validate_presence_of(:password)}
+  end
+
+  describe "association" do
+    it { should have_many(:tour_requests).dependent(:destroy) }
+    it { should have_many(:reviews).dependent(:destroy) }
+  end
+
+  describe "check scope" do
+    let(:user_2){
+      build(:user_example2, email: "khoa2@gmail.com")
```

Comment:
```suggestion
      FactoryBot.create :user_example2, email: "khoa2@gmail.com"
```

fix tương tự những chỗ khác